src/deploymentmanager.ts

```diff
@@ -553,6 +559,17 @@ export class DeploymentManager implements IDeploymentManager {
 
         console.log('Copy the following environment variables to /scripts/local/.env file: \n\ %s', `${chalk.cyan(data.join('\n'))}`);
     }
+
+    // Internal cloud names for diagnostics
+    private getCloudType(environmentName: string): string {
+        const cloudTypeMaps = {
```

Comment:
Do you want to move the const out? Also why can't we send enum.toString() value. Why do we need to do a conversion?

Reply:
The list is provided by internal team for data analysis.